rpm/kubelet.spec

```diff
@@ -0,0 +1,87 @@
+%global KUBE_VERSION 1.3.4
```

Comment:
In the context of a very defined build script yeah quite likely, I think you can do rpmbuild --define 'KUBE_VERSION 1.3.4' and remove the %global here.